latch/types/directory.py

```diff
@@ -196,6 +197,9 @@ def iterdir(self) -> List[Union[LatchFile, "LatchDir"]]:
 
         return ret
 
+    def size(self):
```

Comment:
```suggestion
    def size_recursive(self):
```

probably do this?
want to make it clear that it's nowhere near as fast as normal `.size()`